flink-yarn/src/test/java/org/apache/flink/yarn/YarnClusterDescriptorTest.java

```diff
@@ -741,6 +742,37 @@ public void testDeployApplicationClusterWithDeploymentTargetNotCorrectlySet() {
         }
     }
 
+    @Test
+    public void testGetStagingDirWithoutSpecifyingStagingDir() throws IOException {
+        try (final YarnClusterDescriptor yarnClusterDescriptor = createYarnClusterDescriptor()) {
+            YarnConfiguration yarnConfig = new YarnConfiguration();
+            yarnConfig.set("fs.defaultFS", "file://tmp");
+            FileSystem defaultFileSystem = FileSystem.get(yarnConfig);
+            Path stagingDir = yarnClusterDescriptor.getStagingDir(defaultFileSystem);
+            assertEquals(
```

Comment:
We can be more specific in this case like down below:
```
            assertEquals("file", defaultFileSystem.getScheme());
            assertEquals("file", stagingDir.getFileSystem(yarnConfig).getScheme());
```

Reply:
Done